### PR TITLE
Add models and corresponding specs

### PR DESCRIPTION
**Pull request by chubchenko**

### CODE REVIEW

File: app/models/vertigo/rtm/conversation.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+      has_many   :conversation_user_relations, dependent: :destroy
+      has_many   :members, through: :conversation_user_relations, class_name: Vertigo::Rtm.user_class
+      belongs_to :creator, class_name: Vertigo::Rtm.user_class, foreign_key: :creator_id
+      has_many   :messages, dependent: :destroy
+
+      validates :name, uniqueness: true, presence: true
+
+      def group?
+        false
```
Comment (sparrow) [inline app/models/vertigo/rtm/conversation.rb]: what about `self.class.name.split('::').last == 'Group'` ?

File: app/models/vertigo/rtm/conversation.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      end
+
+      def channel?
+        false
```
Comment (sparrow) [inline app/models/vertigo/rtm/conversation.rb]: and `self.class.name.split('::').last == 'Channel'` ?

**Comment by chubchenko** [comment]
@sparrow fixed

**Comment by kirillshevch** [comment]
+

File: app/models/vertigo/rtm/attachment.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module Vertigo
+  module Rtm
+    class Attachment < ApplicationRecord
+      belongs_to :message
```
Comment (GeorgeZhukov) [inline app/models/vertigo/rtm/attachment.rb:4]: what about validations for message presence?

Comment (chubchenko) [inline app/models/vertigo/rtm/attachment.rb:4]: @GeorgeZhukov Rails 5 makes belongs_to association required by default

Comment (GeorgeZhukov) [inline app/models/vertigo/rtm/attachment.rb:4]: Oops, my fault. Thx.

**Comment by GeorgeZhukov** [comment]
+

**Comment by vshmyhlo** [comment]
:+1:

**Pull request merged**